dspace/config/pages/help.xml

```diff
@@ -2,38 +2,51 @@
 
 <body>
 <div id="file.help.div.help" n="help" rend="primary">
-<head>MOspace Help</head>
-
+<head>MOspace Institutional Repository Help</head>
 
 <div id="top" n="top">
+<body>
```

Comment:
the body tag does not belong here, it's a wrapper for the entire page (see line 3)